samples/net/sockets/echo_client/boards/intel_socfpga_agilex5_socdk.conf

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022-2023, Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_MDIO=y
+CONFIG_NET_L2_ETHERNET=y
+
+CONFIG_NET_PKT_RX_COUNT=64
+CONFIG_NET_PKT_TX_COUNT=64
+CONFIG_NET_BUF_RX_COUNT=1024
+CONFIG_NET_BUF_TX_COUNT=512
+
+CONFIG_ISR_STACK_SIZE=4096
+CONFIG_MAIN_STACK_SIZE=10240
+CONFIG_IDLE_STACK_SIZE=2048
+CONFIG_SHELL_STACK_SIZE=10240
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8192
+CONFIG_NET_MGMT_EVENT_STACK_SIZE=32768
+CONFIG_NET_TCP_WORKQ_STACK_SIZE=4096
+CONFIG_NET_RX_STACK_SIZE=32768
+
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="10.10.0.101"
+CONFIG_NET_CONFIG_MY_IPV4_NETMASK="255.255.255.0"
+CONFIG_NET_CONFIG_MY_IPV4_GW="8.8.8.8"
+CONFIG_NET_CONFIG_PEER_IPV4_ADDR="10.10.0.102"
```

Comment:
Using these values from your lab/test network in this config are not ok. These should not be set per board, as the default ones are set in prj.conf. You can always override the default values when you compile your own application if needed.
Also questioning why Google DNS server 8.8.8.8 is set as a default gateway, this cannot be right?

Reply:
removed these configurations